### DISCUSSION TITLE
Which month do people 💤🛌 the least?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1118] [Galaxy Trivia] [answered]
**Bonus:** What is the world record for the longest deliberate 😴 deprivation?

**Comment by peterbaccaro** [comment, marked as answer]
People sleep the least in **February** because the are usually fewer nights in February.

The world record for the longest deliberate sleep deprivation is **453 hours 40 minutes (18 days 21 hours 40 minutes)**.  This record was achieved by Robert McDonald in 1986.

https://www.guinnessworldrecords.com/news/2023/1/whats-the-limit-to-how-long-a-human-can-stay-awake-733188
